pom.xml

```diff
@@ -220,57 +220,57 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.mortbay.jetty</groupId>
-        <artifactId>jetty-maven-plugin</artifactId>
-        <version>8.1.8.v20121106</version>
-        <configuration>
-          <!--  <scanIntervalSeconds>10</scanIntervalSeconds> -->
-          <stopPort>8000</stopPort>
-          <stopKey>STOP</stopKey>
-          <!--
-              <contextPath>/</contextPath>
-          <webDefaultXml>src/test/resources/webdefault.xml</webDefaultXml>
-          <webXml>src/test/webapp/WEB-INF/web.xml</webXml>
-          <jettyConfig>src/test/resources/jetty.xml</jettyConfig>
-          <webAppSourceDirectory>src/test/webapp</webAppSourceDirectory>
-          <systemProperties>
-            <systemProperty>
-              <name>org.expath.pkg.saxon.repo</name>
-              <value>${basedir}/xrepo</value>
-            </systemProperty>
-          </systemProperties>
-          -->
-        </configuration>
-        <executions>
-          <execution>
-            <id>start-jetty</id>
-            <phase>pre-integration-test</phase>
-            <goals>
-              <goal>start</goal>
-            </goals>
-            <configuration>
-              <scanIntervalSeconds>0</scanIntervalSeconds>
-              <daemon>true</daemon>
-            </configuration>
-          </execution>
-          <execution>
-            <id>stop-jetty</id>
-            <phase>post-integration-test</phase>
-            <goals>
-              <goal>stop</goal>
-            </goals>
-          </execution>
-        </executions>
-        <dependencies>
-          <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.16</version>
-            <type>jar</type>
-          </dependency>
-        </dependencies>
-      </plugin>
+      <!--<plugin>-->
```

Comment:
@lessarderic Blame intelliJ, It automatically did individual elements because i didn't hold down shift or whatever the modifier is.  :p (I Will fix it though)

Moral of the story though java 8 is causing a lot of issues. (I believe its what's causing this plugin from not working as well.) We need to  figure out what we want to do In terms of java version.